publication-commons/src/test/java/no/unit/nva/publication/model/business/publicationchannel/PublicationChannelTest.java

```diff
@@ -190,10 +191,57 @@ void shouldReturnListingResultWhenListingPublicationChannels()
         assertEquals(2000, totalChannelsRetrieved.size());
     }
 
-    private void persistResourcesPublicationChannels(int numberOfResources, URI publisherId, ChannelClaimDto claim,
+    @Test
+    void shouldUpdatePublicationChannelsInBatch() {
+        var channel = randomClaimedPublicationChannel();
+        super.persistPublicationChannel(channel);
+
+        assertDoesNotThrow(() -> resourceService.batchUpdateChannels(List.of(channel)));
+    }
+
+    @Test
+    void shouldUpdateNonClaimedChannelToClaimed() {
+        var channel = randomNonClaimedPublicationChannel();
+        var customerId = randomUri();
+        var organizationId = randomUri();
+        var constraint = randomConstraint();
+
+        var claimedChannel = channel.toClaimedChannel(customerId, organizationId, constraint);
+
+        assertInstanceOf(ClaimedPublicationChannel.class, claimedChannel);
+        assertEquals(customerId, claimedChannel.getCustomerId());
+        assertEquals(organizationId, claimedChannel.getOrganizationId());
+        assertEquals(constraint, claimedChannel.getConstraint());
```

Comment:
Do you expect anything to not be equal? I guess modified date, but maybe nothing substantive beyond type.

Reply:
Yes, everything except modifiedDate :) Will construct expected object